kratos/utilities/auxiliar_model_part_utilities.h

```diff
@@ -273,6 +283,246 @@ class KRATOS_API(KRATOS_CORE) AuxiliarModelPartUtilities
      */
     void RemoveConditionsAndBelongingsFromAllLevels(Flags IdentifierFlag = TO_ERASE);
 
+
+    /// To Export a Scalar data (Double/int/...)
+    template<typename TDataType>
+    std::vector<TDataType> GetScalarData(
+        const Variable<TDataType>& rVariable,
+        const DataLocation DataLoc)
```

Comment:
```suggestion
        const DataLocation DataLoc) const
```

the `Get...` functions should be `const`

If you don't know what it means I will explain in the next meeting as this is an important C++ feature

Reply:
Yes, I understood this. Made it constant, so that this **Get...** method should not modify any of the member variables of the class.